.github/actions/conditional/conditions

```diff
@@ -3,10 +3,10 @@
 # To test a pattern run '.github/actions/conditional/conditional.sh <remote name> <branch>'
 # Alternatively, run 'git diff <remote name>/<branch> <pattern>' (from the root directory)
 
-ci=*/pom.xml */src/main/ */src/test/
+ci=*/pom.xml */src/main/ */src/test/ .github/actions/ .github/workflows/
 
-operator=*/pom.xml */src/main/ */src/test/
+operator=*/pom.xml */src/main/ */src/test/ .github/actions/ .github/workflows/
 
-codeql-java=*.java
-codeql-themes=themes/
-codeql-js_adapter=adapters/oidc/js/
\ No newline at end of file
+codeql-java=*.java .github/workflows/codeql-analysis.yml
```

Comment:
These workflows have dependencies on other files (such as those in `.github/actions`). This means dependent files could be modified without these workflows running.